src/Catalyst.Node.POA.CE.Tests/IntegrationTests/Config/ConfigurableRepoIntegrationTests.cs

```diff
@@ -35,7 +34,7 @@ public sealed class ConfigurableRepoIntegrationTests : FileSystemBasedTest
     {
         public ConfigurableRepoIntegrationTests(ITestOutputHelper output) : base(output) { }
 
-        private async Task ModuleCanSaveAndRetrieveValuesFromRepository(FileInfo moduleFile)
+        private void ModuleCanSaveAndRetrieveValuesFromRepository(FileInfo moduleFile)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'ModuleCanSaveAndRetrieveValuesFromRepository'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)